testing/mm/mm_main.c

```diff
@@ -124,6 +124,17 @@ static struct mallinfo g_alloc_info;
  * Private Functions
  ****************************************************************************/
 
+static bool is_oversize(int size)
+{
+  if (size < 0)
+    {
+      return 0;
+    }
+
+  unsigned long threshold = g_alloc_info.mxordblk * 3 / 4;
+  return threshold < size;
+}
```

Comment:
It feels more natural to return `size > threshold`.
Also return false instead of 0 please.

Reply:
Thanks. Modified according to your comments. @XuNeo